components/ai_chat/core/browser/ai_chat_service.cc

```diff
@@ -279,8 +290,8 @@ ConversationHandler* AIChatService::GetOrCreateConversationHandlerForContent(
     conversation = CreateConversation();
   }
   // Provide the content delegate, if allowed
-  MaybeAssociateContentWithConversation(conversation, associated_content_id,
-                                        associated_content);
+  MaybeAssociateContent(conversation, associated_content_id, associated_content,
```

Comment:
As discussed let's move this to only happen for new conversations

Reply:
Done :smile: